### ISSUE TITLE
Add support for BYWEEKDAY

### ISSUE THREAD

**Issue by jonocodes** [collective/icalendar#268]
dateutils.rrule supports the byweekday parameter already:
http://dateutil.readthedocs.io/en/stable/rrule.html

Could this be added to icalendar?

**Comment by niccokunzmann** [comment]
@natashamm Thanks for working on this issue! I wonder: Do you think that this issue can be closed now? Is there anything else to do?

**Comment by natashamm** [comment]
The other issue I was thinking of is: what if someone generates an RRULE with BYDAY and BYWEEKDAY, do we want to throw an error or automatically replace one with the other? There is nothing in RFC 5545 about that of course since it only considers BYDAY.

**Comment by niccokunzmann** [comment]
@natashamm @jonocodes https://github.com/collective/icalendar/pull/511 was merged and released in v5.0.5. Can you check if this can be closed?

**Comment by natashamm** [comment]
> Since it is an assumption of the algorithm, I would say an assertion would do it.

My bad - do you mean an assumption in https://dateutil.readthedocs.io/en/stable/rrule.html?

**Comment by niccokunzmann** [comment]
No. If we have, these two arguments that mean the same and the value differs, what do we do? Which one is used? rrule can only take one of them, I assume. What are your thoughts?

**Comment by niccokunzmann** [comment]
Ok, thanks! I had a look. It seems that dateutil will handle the parsing of an rrule string and as such the question of how to convert this parameter is not necessary to answer in icalendar. Thanks for the work!
If there is any doubt whether this should be closed, please comment/re-open!
